vm-setup/roles/virtbmc/tasks/setup_tasks.yml

```diff
@@ -149,6 +149,7 @@
       SUSHY_EMULATOR_VMEDIA_VERIFY_SSL = {{ sushy_vmedia_verify_ssl }}
       SUSHY_EMULATOR_AUTH_FILE = "/root/sushy/htpasswd"
       SUSHY_EMULATOR_FAKE_DRIVER = True
+      EXTERNAL_NOTIFICATION_URL = "https://localhost:9999"
```

Comment:
Could you please give this a a name that more explicitly ties the variable to FAKE-IPA ?

Reply:
That's in the sushy-tools and I wanted to be explicit using EXTERNAL_NOTIFICATION prefix it can be fake-ipa or any external component